pyproject.toml

```
@@ -5,13 +5,12 @@ description = ""
 authors = ["CFPB's RegTech Team"]
 readme = "README.md"
 packages = [
-    { include = "regtech_mail_api" }
+    { include = "src/regtech_mail_api" }
```

We've done the other projects like...

```suggestion
    { include = "src/regtech_mail_api" }
    { include = "regtech_mail_api" from = "src"}
```

That way it knows the top-level module is in `src`, but it keep any additional files that're in `regtech_mail_api`, but not in `src`. Not an issue for this particular repo, but might as well keep it consistent.

---

Consistency is good.  I saw that in the others but wasn't sure the difference between one vs the other.  Updated